### DISCUSSION TITLE
Greenfield API: Add storeId to the response of every invoice call

### DISCUSSION THREAD

**Discussion by woutersamaey** [btcpayserver/btcpayserver#2591] [Ideas / Feature Requests]
When developing the Zapier integration I ran into something that's a bit annoying and may be worth considering.

I've noticed that every invoice API responce does NOT include a field for the store ID. The store ID is always in the request URL, but never in the response.

Because it's in the URL, the client knows it, but since only responses are moved around in Zapier (and possibly other tools as well) it proves to be an inconvenience.

Is this something that the team might consider adding or is there an argument against it.

I've solved the issue for now in a hacky way, so not blocking, but maybe worth considering?

**Comment by Kukks** [comment]
@dennisreimann  and I also discussed this and found a need for it. ACK from my end!

**Comment by Kukks** [comment]
https://github.com/btcpayserver/btcpayserver/pull/2592
